file_migration.go

```
@@ -105,8 +106,8 @@ func migrationFromFileFS(name string, dir fs.FS, log *slog.Logger) FileMigration
 // applyStepsStream executes database migration steps read from an io.Reader, separated by semicolons, in a transaction.
 // Returns the corresponding error if any step execution fails. Also, as some database drivers or engines seem to not
 // support comments, leading comments are removed. This function does not undertake efforts to scan the SQL to find
-// other comments. So leading comments telling what a step is going to do, work. But comments in the middle of a
-// statement will not be removed. At least with SQLite this will lead to hard to find errors.
+// other comments. Such leading comments telling what a step is going to do, work. But comments in the middle of a
+// statement will not be removed. At least with SQLite this will lead to hard-to-find errors.
 func applyStepsStream(tx *sql.Tx, r io.Reader, id string, log *slog.Logger) error {
```

golangci-lint (2.2.2)

parameter name 'tx' is too short for the scope of its usage

(varnamelen)